server/views/partials/header.njk

```diff
@@ -1,45 +1,39 @@
         </svg>
-        HMPPS
       </a>
-
-      <a class="hmpps-header__link hmpps-header__title__service-name" data-qa="header-text" href="/">{{ applicationName }}</a>
     </div>
-
-    <nav aria-label="Account navigation">
-      <ul class="hmpps-header__navigation">
-        {% if user %}
-          <li class="hmpps-header__navigation__item">
-            <a data-qa="manageDetails" class="hmpps-header__link" href="/account-details" data-test="manage-account-link" target="_blank" rel="noopener noreferrer">
-              <span data-qa="header-user-name">{{ user.displayName | initialiseName }}</span>
-              <span class="hmpps-header__link__sub-text">Manage your details</span>
-            </a>
-          </li>
-
-          <li class="hmpps-header__navigation__item">
-            <a data-qa="signOut" class="hmpps-header__link" href="/sign-out">Sign out</a>
-          </li>
-        {% endif %}
-
+    <div class="govuk-header__content">
+      <a href="/" class="govuk-header__link govuk-header__service-name">
```

Comment:
can we add data-qa="header-text" to this tag as a fix for a failing integration test.